recipe/meta.yaml

```diff
@@ -16,17 +16,23 @@ build:
     - vc9     # [win and py27]
     - vc10    # [win and py34]
     - vc14    # [win and py35]
+    - vc14    # [win and py36]
+  skip: true  # [win and py35]
 
 requirements:
   build:
+    - krb5 1.14.*
+    - libssh2 1.8.*
     # python is a build requirement on Windows to resolve features.
     - python           # [win]
     # perl is required to run the tests on UNIX.
-    - perl             # [unix]
+    - perl 5.22.2.1    # [unix]
     - pkg-config       # [unix]
     - openssl 1.0.*    # [unix]
     - zlib 1.2.*
   run:
+    - krb5 1.14.*
+    - libssh2 1.8.*
```

Comment:
So I guess we don't have these in the [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/53f174da54152b476abaca28ca85163f1d7e2471/scripts/pin_the_slow_way.py#L40-L81 ) currently. Could you please submit a PR to add them? Would also be good to double check pinnings against the [ABI Laboratory Tracker]( https://abi-laboratory.pro/tracker/ ).